### ISSUE TITLE
Side by side page ，select one side lead html file also preserve the layout

### ISSUE THREAD

**Issue by inevity** [gildas-lormeau/SingleFile#107]
**Describe the bug**
Side by side page, ， such as left is markdown ，right is preview page 
**To Reproduce**

**Expected behavior**

The opened html should occupy whole page,no half.
**Screenshots**

**Environment**
 - OS: macos
 - Browser: Chrome
 - Version: N/A

**Additional context**

**Comment by gildas-lormeau** [comment]
Do you mean that when you save for example https://markdownlivepreview.com/, you expect to see only the HTML on the right but displayed in full width?

**Comment by gildas-lormeau** [comment]
I'm closing this issue because I do not understand it. Feel free to re-open it and give more details.

**Comment by inevity** [comment]
Yes, if i seletion the right preview and save selection. When open the create  html, it only occupy the half width.

**Comment by gildas-lormeau** [comment]
Thank you for the details. I understand the issue now. Actually, the "Save selection" does its best to preserve the original layout of the page otherwise most of the time, it could be ugly (because of missing fonts, font size etc.) or not readable because the layout would be broken. 

From my point of view, you want a "Save selection" feature that would act like the reader mode by removing all the superficial styles. Unfortunately, this problem is hard to solve in a generic way.
